tcl/org.testeditor.tcl.dsl.ui/src/org/testeditor/tcl/dsl/ui/contentassist/TclProposalProvider.xtend

```diff
@@ -33,6 +36,27 @@ class TclProposalProvider extends AbstractTclProposalProvider {
 		acceptor.accept(createCompletionProposal('Component: ', 'Component:', getImage(model), context))
 	}
 
+	override completeTestCase_Steps(EObject model, Assignment assignment, ContentAssistContext context,
```

Comment:
The formatting is not nice plus I think the proposals should depend on the concrete position.